package.json

```diff
@@ -9,6 +9,7 @@
     "skyux": "skyux",
     "start": "skyux serve --hmr",
     "test": "skyux test --coverage library --logFormat none",
+    "watch": "skyux watch",
```

Comment:
Thanks for adding this! We'll need to suffix the command with `--coverage library`:

```
skyux watch --coverage library
```

Reply:
Good call, I'll add that